zerver/lib/test_runner.py

```diff
@@ -28,12 +28,11 @@
 import os
 import time
 import unittest
-import ujson
-import glob
 
-from multiprocessing.sharedctypes import Synchronized
+from multiprocessing.sharedctypes import Synchronized, Array
 
 _worker_id = 0  # Used to identify the worker process.
+_subsuite_index_to_pid = Array('i', 1000)  # Used to reference subsuites by pid
```

Comment:
I'm unsure of the correct approach regarding the size of the array.

Reply:
There’s no need to construct the array so early. Construct it in `Runner.run_tests` with length `len(suite.subsuites)`.

Reply:
So it seems mypy really complains about not assigning this name early on.

Reply:
MyPy is weird sometimes. Easy to work around, though: make an array of size 0 early on, then reassign it later.